app/src/pages/inside/common/utils.js

```diff
@@ -165,7 +165,11 @@ export const getGroupedDefectTypesOptions = (
         color: defectType.color,
         locator: defectType.locator,
         meta: {
+          typeRefLabel: defectTypesLocalization[defectType.typeRef.toLowerCase()]
+            ? formatMessage(defectTypesLocalization[defectType.typeRef.toLowerCase()])
+            : defectType.longName,
```

Comment:
Why not the same as [here](https://github.com/reportportal/service-ui/pull/3286/files#diff-f6ff9dfcbc33e1cec20a1b5176d347ecc945f05f76288ed6d1aaa699d73eb2f2R162-R164)?